sycl/include/sycl/backend_types.hpp

```diff
@@ -11,9 +11,9 @@
 #include <sycl/detail/defines.hpp>
 
 #include <fstream>
-#include <iostream>
 #include <istream>
 #include <string>
+#include <sycl/detail/iostream_proxy.hpp>
```

Comment:
Here and in other files: can we move this include to the `sycl/*` group. I.e., in this file move it between lines 11 and 12.